specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json

```diff
@@ -1796,25 +1798,67 @@
       "type": "object",
       "required": [
         "relationType",
-        "bidirectional",
-        "source",
-        "target"
+        "entities"
       ],
       "properties": {
         "relationType": {
           "description": "Type of relation. Examples include: `DosageOfMedication` or 'FrequencyOfMedication', etc.",
           "type": "string"
         },
-        "bidirectional": {
-          "description": "If true the relation between the entities is bidirectional, otherwise directionality is source to target.",
-          "type": "boolean"
-        },
-        "source": {
-          "description": "Reference link to the source entity.",
+        "entities":{
+          "description": "The entities in the relation.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/HealthcareRelationEntity"
+          }
+        }
+      }
+    },
+    "HealthcareAssertion": {
+      "type": "object",
+      "properties": {
+        "conditionality": {
+          "description": "Describes any conditionality on the entity. Possible values: conditional, hypothetical",
+          "type": "string",
+          "enum": [
+            "Hypothetical",
+            "Conditional"
+          ]
+        },        
+        "certainty": {
+          "description": "Describes the entities certainty and sentiment. Possible values: positive, positive possible, neutral, negative possible, negative",
+          "type": "string",
+          "enum": [
+            "Positive",
+            "Positive Possible",
+            "Neutral Possible",
+            "Negative Possible",
+            "Negative"
+          ]
+        },
+        "association": {
+          "description": "Describes if the entity is the subject of the text or if it describes someone else. Possible values: subject, other",
+          "type": "string",
+          "enum": [
+            "subject",
+            "other"
+          ]
+        }
+      }
+    },
+    "HealthcareRelationEntity": {
+      "type": "object",
+      "required": [
+        "ref",
+        "role"
+      ],
+      "properties": {
+        "ref": {
+          "description": "Reference link to the entity object.",
           "type": "string"
         },
-        "target": {
-          "description": "Reference link to the target entity.",
+        "role": {
+          "description": "Role of entity in the relationship. For example: 'CD20-positive diffuse large B-cell lymphoma' has the following entities with their roles in parenthesis:  CD20 (GeneOrProtein), Positive (Expression), diffuse large B-cell lymphoma (Diagnosis).",
```

Comment:
The common use case is usually simply an `Attribute` and `Entity`, right?

Reply:
Correct, the most common use case is Attribute and Entity